CSRF_Protect.php

```diff
@@ -68,11 +68,25 @@ public function echoInputField()
 	 */
 	public function verifyRequest()
 	{
-		if (!isTokenValid($_POST[$this->namespace]))
+		if (!$this->isValidRequest())
 		{
 			die("CSRF validation failed.");
 		}
 	}
+
+	/**
+	 * Verifies whether the post token was set, and return true or false
+	 */
+	public function isValidRequest()
+	{
+		if(!isset($_POST[$this->namespace]))
+			return false;
+		
+		if (!$this->isTokenValid($_POST[$this->namespace]))
```

Comment:
I think this line can just be `return $this->isTokenValid($_POST[$this->namespace]`